### ISSUE TITLE
Commands time out when terminal is focused

### ISSUE THREAD

**Issue by sbungartz** [pokey/command-server#14]
Whenever the built in terminal of VS Code has focus, commands issued using `user.vscode` are no longer working. Instead you get a time out in talon log.

For example:
1. Open a VS Code window and focus the built in terminal.
2. Say `please`
3. Nothing happens, except the following error showing up in talon log:
```
talon.scripting.talon_script.TalonScriptError: 
 in script at /home/simon/.talon/user/knausj_talon/apps/vscode/vscode.talon:16:
 > user.vscode('workbench.action.showCommands')
Exception: Timed out waiting for response
```

I am on
- Ubuntu 20.04
- talon beta v0.3.1-41-g7516 (41)
- knausj_talon 644d330be80bddfe7bfc314a15903185b3111cf7
- VS Code 1.72.2
- Command Server Plugin v0.8.2

Maybe this is a Linux thing, but I have no idea why focusing the terminal would stop command server from reading files...

**Comment by AndreasArvidsson** [comment]
Weird. This appears to be working fine on windows.

**Comment by sbungartz** [comment]
I have found a way to make this work. After adding `command-server.runCommand` to the list of commands whose keybindings will always be handled by VS Code (`terminal.integrated.commandsToSkipShell`), it works!

![image](https://user-images.githubusercontent.com/7670818/197597602-f1c1d13e-98e9-4f35-b4d8-8efeaa85a3f4.png)

Otherwise the key combination seems to be handled by the terminal.

Do you know if there is a way for this plugin to insert itself into this setting automatically?

**Comment by pokey** [comment]
Yes that's the one; couldn't remember the setting name. Glad you were able to find it

Yes I believe there is a way to do that. Settings can contribute default values, and if that doesn't work they can actually just modify settings directly

**Comment by frosk1** [comment]
had the same issue and it could be resolved with adding "command-server.runCommand" to terminal.integrated.commandsToSkipShell like @sbungartz

**Comment by ProfDoof** [comment]
Hey, are you all talking about this setting? [`contributes.configurationDefaults`](https://code.visualstudio.com/api/references/contribution-points#contributes.configurationDefaults)

If so, I might try to spin up a quick PR

**Comment by AndreasArvidsson** [comment]
Good detective work guys. I have created a pull request to try to fix this.
